### ISSUE TITLE
Site shows as "Unsafe" in Edge / Windows

### ISSUE THREAD

**Issue by gndelia** [hemilabs/interface#47]
Users accessing the site in Windows using the Edge browser get a warning about the site being unsafe

![image](https://github.com/user-attachments/assets/3279a480-6a66-4498-bf25-a492660df542)

**Comment by rmilrad** [comment]
@gndelia @daniel-bloq is this issue still active?

**Comment by gndelia** [comment]
Yes, @Jdbiggs is working on it (I can't assign it to him, I think he doesn't belong to the org)

EDIT: now I was able to do it 😄

**Comment by daniel-bloq** [comment]
Here is how MSFT says to fix this issue. I am looking into what I should use to scan the site this morning. 

If your website is blocked by Windows Defender and reported as phishing, you first need to ensure that your website is secure and does not contain any malicious software or links. You can solve this problem by following these steps:

Scan your website: Use security tools or online services to scan your website and ensure that there are no malicious codes or vulnerabilities.

Clean up malicious content: If you find that your website contains malicious content or links, please clean them up immediately.

Update security measures: Ensure that your website software and plugins are the latest version and that necessary security measures have been taken, such as using firewalls, encrypted connections, etc.

Submit for Review: Once you confirm that your website has been cleared of malicious content and is secure, you can submit your website for review so that Windows Defender can remove it from the blacklist.

Request Review: You can contact the support team of Windows Defender and request them to review your website, explain the measures you have taken, and request them to remove your website from the blacklist.

By following the above steps, you can clear the warning and restore customer access to your website. But please make sure your website remains secure at all times and conduct regular security checks to prevent similar issues from happening again.

**Comment by gabmontes** [comment]
@daniel-bloq any update on this?
